client/controllers/events/eventAffectedAreas.coffee

```diff
@@ -153,7 +153,8 @@ Template.eventAffectedAreas.onRendered ->
     EventIncidents.find(@data.filterQuery.get()).map (incident) ->
       incident.locations.forEach (location) ->
         key = "#{location.latitude},#{location.longitude}"
-        incidentsByLatLng[key] = (incidentsByLatLng[key] or []).concat(incident)
+        if not incidentsByLatLng[key]
```

Comment:
This event is the only that I've found that has this issue (I guess the only event that has incidents which overlap): https://eidr-connect.eha.io/events/curated-events/gfnPs88SBb3aaBeeA/affected-areas
On the "Affected Areas" tab click incidents to reveal the markers. The warnings should appear in the console.